save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt

```diff
@@ -280,7 +361,8 @@ class SandboxView : AbstractView<SandboxViewProps, SandboxViewState>(true) {
             |}
         """.trimMargin()
         private val setupShExample = """
-            |python3.10 -m pip install pylint
+            |# Here you can add some additional commands required to run your tool e.g.
+            |# python3.10 -m pip install pylint
```

Comment:
Why do we have a fixed version of python here? Maybe simply `python` would be more generic?

Reply:
Sure

Reply:
Done